packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -584,6 +584,32 @@ export const RemoteServerFactory = {
         app.use(ssrRequestProcessorMiddleware)
     },
 
+    _maintenanceMiddleware(app) {
+        process.env.MRT_MAINTENANCE_MODE = 'no'
```

Comment:
`process.env.MRT_MAINTENANCE_MODE` is supposed to come from Managed Runtime. This is just for testing purposes